src/renderer/helpers/utils.js

```diff
@@ -5,6 +5,8 @@ import FtToastEvents from '../components/ft-toast/ft-toast-events'
 import i18n from '../i18n/index'
 import router from '../router/index'
 
+import { isNullOrEmpty } from './strings'
+
```

Comment:
Looks like the linter warning about this import being unused in this file is correct.
```suggestion

```

Reply:
WIP change that I forgot to remove before committing
Removed now